docs/repositories-configure/integrations/github-integration.md

```diff
@@ -48,3 +50,18 @@ Adds annotations on the lines of the pull request where Codacy finds new issues.
 Shows an overall view of the changes in the pull request, including new issues and metrics such as complexity and duplication. To enable this option, you must enable **Status checks** first.
 
 ![Pull request summary on GitHub](images/github-integration-pr-summary.png)
+
+### Suggest fixes
+
+Adds comments on the lines of the pull request where Codacy finds new issues with suggestions on how to fix the issues. Codacy doesn't apply any changes automatically. To apply the changes, manually review and accept the suggestions.
+
+<!-- TODO Add screenshot -->
+
+!!! note
+    This feature is in early access and has the following limitations:
+
+    -   The only supported Git providers are GitHub Cloud and GitHub Enterprise
```

Comment:
Do we want to specify the minimum enterprise version with it? Should we update with GitHub Enterprise only after we make a release with it?

Reply:
I prefer not to mention the version when a new feature becomes available on the documentation because this creates a maintenance problem and adds unnecessary "noise" for someone who is already on the latest version (such as all our Codacy Cloud users). The way we can let our customers know when we make changes to the product is through the [release notes](https://docs.codacy.com/release-notes/).

However, your concern about the feature becoming available on Codacy Self-hosted is very valid. As it is now, although we'll publish this change on the **Latest** documentation version as soon as we release the feature, we'll only update the documentation version that applies to Codacy Self-hosted on the next Self-hosted release.

(Things get can get more tricky if by chance we release the feature on Codacy Cloud and then release a Self-hosted version that doesn't include the feature. But until now we've always included the latest changes from Codacy Cloud, as long as they apply to Codacy Self-hosted, right?)